src/main/java/seedu/address/logic/parser/ListCommandParser.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ListCommand object.
+ */
+public class ListCommandParser implements Parser<ListCommand> {
```

Comment:
Can use a matcher instead to check for the patients or appts words in the user input